src/components/TimezoneSelect.vue

```diff
@@ -49,12 +49,21 @@
   onUnmounted(() => clearTimeout(timeout))
 
   const timezones = Intl.supportedValuesOf('timeZone').map(timezone => ({ label: timezone, value: timezone }))
-  const options: SelectOption[] = [
-    { label: 'UTC', value: utcTimezone },
-    ...timezones,
+  const options: SelectOptionGroup[] = [
+    {
+      label: 'Suggested timezones',
+      options: [{ label: 'UTC', value: utcTimezone }],
+    },
+    {
+      label: 'All timezones',
+      options: [...timezones],
+    },
   ]
 
+  const localTimezone = Intl.DateTimeFormat().resolvedOptions().timeZone
   if (!props.hideUnset) {
-    options.unshift({ label: 'Use browser default', value: null })
+    options[0].options.unshift({ label: 'Use browser default', value: null })
+  } else {
+    options[0].options.unshift({ label: localTimezone, value: localTimezone })
   }
```

Comment:
This isn't going to be reactive to `props.hideUnset` and its also a little hard to read. I think making `options` a computed would solve both.